bin/docker-build

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+JQ_QUERY='.repositories[] | select(.repositoryUri == "'"$DOCKER_IMAGE"'") | .repositoryName'
+ECR_REPOSITORY_NAME="$(aws ecr describe-repositories | "$JQ_PATH" -r "$JQ_QUERY")"
+[ -n "$ECR_REPOSITORY_NAME" ] || ( >&2 echo "Can't find the ECR repository."  && exit 1 )
+
+JQ_QUERY='.imageDetails[] | select(.imageTags[]? | . == "'"$TAG"'")'
+if aws ecr describe-images --repository-name "$ECR_REPOSITORY_NAME" | "$JQ_PATH" -e "$JQ_QUERY"; then
+  exec echo Skip image building because it has already been built.
```

Comment:
Lets make this an explicit exit so those with less bash-foo know what's going on.